docs/examples/2_urban_scale/index.md

```diff
@@ -0,0 +1,289 @@
+
+Technologies can inherit from anything defined in `tech_groups`, while nodes can inherit from anything in `node_groups`.
+items in `[tech/node]_groups` can also inherit from each other, so you can create inheritance chains.
+
+`interest_rate_setter` looks like this:
+
+```yaml
+--8<-- "src/calliope/example_models/urban_scale/model_config/techs.yaml:interest-rate-setter"
+```
+
+### Conversion technologies
+
+The example model defines two conversion technologies.
+
+The first is `boiler` (natural gas boiler), which serves as an example of a simple conversion technology with one input carrier and one output carrier.
+Its only constraints are the cost of built capacity (`costs.monetary.flow_cap`),
+a constraint on its maximum built capacity (`constraints.flow_cap_max`),
+and a carrier conversion efficiency (`flow_out_eff`).
+
+<figure>
+<img src="../../img/conversion.svg", width="100%", style="background-color:white;", alt="The layout of a conversion technology which has one carrier input and output">
+<figcaption>The layout of a simple conversion technology, in this case `boiler`, which has one carrier input and output, a carrier conversion efficiency ($flow_{eff}^{out}$),
+and a constraint on its maximum built $flow_{cap}$ (which puts an upper limit on $flow_{out}$).</figcaption>
+</figure>
+
+The definition of this technology in the example model's configuration looks as follows:
+
+```yaml
+--8<-- "src/calliope/example_models/urban_scale/model_config/techs.yaml:boiler"
+```
+
+There are a few things to note.
+First, `boiler` defines a name and a color (given as an HTML color code).
+These can be used when visualising your results.
+Second, it specifies its parent, `conversion`, its inflow carrier `gas`, and its outflow carrier `heat`, thus setting itself up as a gas to heat conversion technology.
+This is followed by the definition of constraining parameters and costs;
+the only cost class used is monetary but this is where other "costs", such as emissions, could be defined.
+
+The second technology is `chp` (combined heat and power), and serves as an example of a possible conversion_plus technology making use of two output carriers.
+
+<figure>
+<img src="../../img/conversion_plus_chp.svg", width="100%", style="background-color:white;", alt="The layout of a more complex node which makes use of multiple output carriers.">
+<figcaption>The layout of a more complex node which makes use of multiple output carriers.</figcaption>
+</figure>
+
+This definition in the example model's configuration is more verbose:
+
+```yaml
+--8<-- "src/calliope/example_models/urban_scale/model_config/techs.yaml:chp"
+```
+
+Again, `chp` has the definitions for name, color, parent, and carrier_in/out.
+It has two carriers defined for its outflow.
+The _custom_ parameter `heat_to_power_ratio` is used to create a link between the two output carriers.
```

Comment:
```suggestion
Note the parameter `heat_to_power_ratio`, which we set to 0.8.
We will use this to create a link between the two output carriers.
More importantly, it is a _custom parameter_ - Calliope itself does not define `heat_to_power_ratio` and therefore, for now, it will not have any effect. We need to introduce our own custom math, to which we will turn our attention next.
```